deps/ccommon/src/cc_mm.c

```diff
@@ -34,7 +35,7 @@
 #endif
 
 void *
-_cc_alloc(size_t size, const char *name, int line)
+_cc_alloc(size_t size, bool init, const char *name, int line)
```

Comment:
I'm fairly certain that malloc()/free() is already intercepted by VTune, making this unnecessary.
And on top of that, we really shouldn't introduce additional parameters into alloc().

Reply:
The intention of this change, was to distinguish situations, when the memory region would be initialized ( e.g. by memset call).

In fact this is parameter (_initialized_) used in Identify allocation functions in Vtune API ( see https://software.intel.com/en-us/inspector-user-guide-linux-apis-for-custom-memory-allocation )

Without this change it would be not possible to distnguish it ( see that before my change _cc_zalloc call _cc_alloc and then call memset) and I don't want it to call it twice for it.